### PR TITLE
Add ADME Excretion from tdc

### PR DESCRIPTION
**Pull request by phalem**
Add:
Half Life, Obach et al.
Clearance, AstraZeneca
from:
https://tdcommons.ai/single_pred_tasks/adme/#excretion

### CODE REVIEW

**Review by phalem — COMMENTED** [review]
Add benchmark field

**Comment by kjappelbaum** [comment]
I think there are still issues with the pre-commit. For the bibliography long lines, you might just ignore the checker by adding ` # noqa`

File: data/clearance_astrazeneca/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  Drug clearance is defined as the volume of plasma cleared of a drug
+  over a specified time period and it measures the rate at which the active drug
+  is removed from the body. This is a dataset curated from ChEMBL database containing
+  experimental results on intrinsic clearance, deposited from AstraZeneca. It
+  contains clearance measures from two experiments types, hepatocyte and microsomes.
+targets:
+- id: drug_clearance
+  description: The volume of plasma cleared of a drug over a specified time period
+    and it measures the rate at which the active drug is removed from the body
+  units: mL.min-1.g-1
```
Comment (kjappelbaum) [inline data/clearance_astrazeneca/meta.yaml:12]:
```suggestion
  units: mL / (min  g)
```

File: data/clearance_astrazeneca/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  - Pharmacokinetics drug clearance
+  - Drug Excretion
+  - microsome/hepatocyte clearance
+  uris:
+  - https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMESH%2FD006207
+  - https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C94618
+benchmarks:
+- name: TDC
+  link: https://tdcommons.ai/
+  split_column: split
```
Comment (kjappelbaum) [inline data/clearance_astrazeneca/meta.yaml:27]: did you add this in the CSV?

File: data/clearance_astrazeneca/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  type: continuous
+  names:
+  - Drug clearance
+  - The rate at which the active drug is removed from the body
+  - ADME Drug clearance
+  - Pharmacokinetics drug clearance
+  - Drug Excretion
+  - microsome/hepatocyte clearance
+  uris:
+  - https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMESH%2FD006207
```
Comment (kjappelbaum) [inline data/clearance_astrazeneca/meta.yaml:22]: is this dataset actually looking at the half life?

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
Thanks again! You did most the PRs here! 
Again similar comments are the other ones.

File: data/clearance_astrazeneca/meta.yaml (hunk 3 above)
Comment (phalem) [reply data/clearance_astrazeneca/meta.yaml:22]: In a pharmacological manner yes

**Comment by phalem** [comment]
> Thanks again! You did most the PRs here! Again similar comments are the other ones.
Thanks Kevin for your kind word. I hope that I can finish it. Yeah, I found that I miss the split on the trasnfrom.py in all new PR. So, I will add it. Sorry, for this mistakes.

File: data/clearance_astrazeneca/meta.yaml (hunk 1 above)
Comment (MicPie) [reply data/clearance_astrazeneca/meta.yaml]: changed

File: data/clearance_astrazeneca/meta.yaml (hunk 3 above)
Comment (MicPie) [reply data/clearance_astrazeneca/meta.yaml]: I removed this from the `clearance_astrazeneca` dataset

File: data/clearance_astrazeneca/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: chembl_id
+      type: Other
+      names:
+          - ChEMBL id
+          - ChEMBL identifier number
+      description: ChEMBL ids
```
Comment (kjappelbaum) [inline data/clearance_astrazeneca/meta.yaml:32]: Also here I wonder about the ID. Do we need an attribute that prevents sampling this column?

File: data/half_life_obach/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: chembl_id
+      type: Other
+      names:
+          - ChEMBL database id
+          - ChEMBL identifier number
+      description: ChEMBL ids
```
Comment (kjappelbaum) [inline data/half_life_obach/meta.yaml:31]: see above

File: data/clearance_astrazeneca/meta.yaml (hunk 4 above)
Comment (MicPie) [reply data/clearance_astrazeneca/meta.yaml:32]: If we always use `chembl_id` as col id we could simply exclude in our sampling logic?

Comment (MicPie) [reply data/clearance_astrazeneca/meta.yaml:32]: or we simply comment out those lines so we have it should we need them again?

Comment (MicPie) [reply data/clearance_astrazeneca/meta.yaml:32]: added separate flag

File: data/half_life_obach/meta.yaml (hunk 5 above)
Comment (MicPie) [reply data/half_life_obach/meta.yaml:31]: added separate flag for it

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**